src/main/java/FrameworksDrivers/LogInView.java

```diff
@@ -17,15 +17,18 @@ public class LogInView implements ActionListener, View {
     Label userLabel, passLabel;
     textField textFieldUser, textFieldPass;
 
+    CardLayout layout;
+
     private Object[] paths;
 
-    public LogInView(JPanel masterPanel){
+    public LogInView(JPanel masterPanel, CardLayout layout){
         //create page to display created element
         this.masterPanel = masterPanel;
+        this.layout = layout;
 
         this.newPanel = new JPanel();
```

Comment:
I see here that you are intializing the newPanel attribute but it isnt listed as an attribute in the class. Technically it isn't necessary to list all the attributes but it is best that you do so that its clear that it exists